tests/slack_sdk/web/mock_web_api_server.py

```diff
@@ -70,6 +92,12 @@ def _handle(self):
                 if self.headers["authorization"] == "Basic MTExLjIyMjpzZWNyZXQ=":
                     self.wfile.write("""{"ok":true}""".encode("utf-8"))
                     return
+                elif (
+                    self.headers["authorization"]
+                    == "Basic MTExLjIyMjp0b2tlbl9yb3RhdGlvbl9zZWNyZXQ="
```

Comment:
This means the values (client_id="111.222", client_secret="token_rotation_secret") that are used in the tests